src/FirmwarePlugin/PX4/PX4FirmwarePlugin.cc

```diff
@@ -174,7 +174,7 @@ bool PX4FirmwarePlugin::setFlightMode(const QString& flightMode, uint8_t* base_m
 
 int PX4FirmwarePlugin::manualControlReservedButtonCount(void)
 {
-    return 8;   // 8 buttons reserved for rc switch simulation
+    return 0;   // 0 buttons reserved for rc switch simulation
```

Comment:
If you look here: https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_receiver.cpp#L1089. You'll see that the button bits are turned into RC channels. This is the reason why this is 8. The possible solution is to stop sending button bits in the MANUAL_CONTROL message. Not sure of the value of turning single position joystick switches into rc channels. The results can be unexpected depending on how you have flight modes set up. @LorenzMeier ?